katcp/server.py

```diff
@@ -2264,6 +2264,7 @@ class DeviceLogger(object):
         WARN: logging.WARN,
         ERROR: logging.ERROR,
         FATAL: logging.FATAL,
+        OFF: logging.CRITICAL + 10 # OFF is the highest possible logging level
```

Comment:
logging.CRITICAL and logging.FATAL are the same thing. But it would make more sense in this file if you stick to the one or the other. since FATAL is used every where rather reference that here.
OFF: logging.FATAL + 10